src/PerfectGym.AutomergeBot/SlackNotifications/SlackMessageProvider.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Text;
+
+namespace PerfectGym.AutomergeBot.SlackNotifications
+{
+    public class SlackMessageProvider : ISlackMessageProvider
+    {
+        private readonly INow _now;
+
+        public SlackMessageProvider(INow now)
+        {
+            _now = now;
+        }
+
+        public string CreateNotifyUserAboutPendingPullRequestMessage(string authorId,
```

Comment:
Wydaje mi się, że lepiej, gdy wszystkie parametry są wypisane w jednej kolumnie. Tzn. 
```suggestion
        public string CreateNotifyUserAboutPendingPullRequestMessage(
            string authorId,
```

Reply:
ok